sway-lib-std/src/token.sw

```diff
@@ -41,7 +40,7 @@ use ::outputs::{Output, output_amount, output_count, output_type};
 /// ```
 pub fn mint_to(to: Identity, sub_id: SubId, amount: u64) {
     mint(sub_id, amount);
-    transfer(to, sha256((contract_id(), sub_id)), amount);
+    transfer(to, AssetId::new(contract_id(), sub_id), amount);
```

Comment:
Same comments here as for the call below.

Reply:
What call above? What are you referring to?

Reply:
https://github.com/FuelLabs/sway/pull/4955#discussion_r1306043424